test/module-index.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
```

Comment:
plz remove unused imports

Reply:
Ah, good call. A linter would be helpful to catch these. If I may suggest as a side note, [eslint](https://eslint.org/) would be the way to go.